src/stories/charts/sentiment/index.tsx

```diff
@@ -0,0 +1,268 @@
+import { ResponsiveLine, SliceTooltipProps } from "@nivo/line";
+import { SentimentChartProps } from "./_types";
+import {
+  DEFAULT_CHARTS_THEME,
+} from "../../theme/charts";
+import { ChartContainer } from "../ChartContainer";
+import styled, { ThemeContext } from "styled-components";
+import { useContext } from "react";
+import { ReactComponent as S1 } from "./assets/sentiment_1.svg";
+import { ReactComponent as S2 } from "./assets/sentiment_2.svg";
+import { ReactComponent as S3 } from "./assets/sentiment_3.svg";
+import { ReactComponent as S4 } from "./assets/sentiment_4.svg";
+import { ReactComponent as S5 } from "./assets/sentiment_5.svg";
+import { DatumValue } from "@nivo/core";
+import { getColor } from "../../theme/utils";
+
+const Point = styled.g`
+  transform: translate(-13px, -13px);
+`;
+
+const ScrollingContainer = styled.div`
+  width: 100%;
+  overflow-x: scroll;
+`;
+
+const SentimentContainer = styled(ChartContainer)`
+  /* Show dotted lines */
+  svg > g > g:nth-child(2) > line {
+    stroke-dasharray: 2;
+  }
+
+  /* Show first and last vertical lines */
+  svg > g > g:first-child > line:first-child {
+    transform: translate(2px, 0);
+    stroke-dasharray: 2;
+  }
+
+  svg > g > g:first-child > line:last-child {
+    transform: translate(-2px, 0);
+    stroke-dasharray: 2;
+  }
+
+  /* Show first and last horizontal lines */
+  svg > g > g:nth-child(2) > line:first-child {
+    transform: translate(0, -6px);
+  }
+
+  svg > g > g:nth-child(2) > line:last-child {
+    transform: translate(0, 2px);
+  }
+`;
+
+const DEFAULT_CHART_MARGINS = { top: 0, right: 0, bottom: 30, left: 30 };
+
+const formatSentiment = (value: DatumValue) => {
```

Comment:
questo formatSentiment come gestisce le traduzioni?conviene aggiungere il parametro i18n

Reply:
giusto, va anche messa la traduzione per "Neutral" la linea mediana

Reply:
in realtà le label per i sentiment non sono più usate in seguito alle modifiche sul design del componente, ma manterrei comunque la funzione per tutti i valori dei sentiment anche se attualmente è usato solo "neutral"